casper-finality-proofs/src/test_engine/wrappers/wrapper_compute_shuffled_index.rs

```diff
@@ -0,0 +1,40 @@
+use crate::assert_equal;
+use crate::compute_shuffled_index::compute_shuffled_index::ComputeShuffledIndex;
+use crate::test_engine::types::compute_shuffled_index_data::TestData;
+use crate::test_engine::utils::parse_file::read_fixture;
+use once_cell::sync::Lazy;
+use plonky2x::backend::circuit::MockCircuitBuild;
+use plonky2x::prelude::{Bytes32Variable, U64Variable};
+use plonky2x::prelude::{CircuitBuilder, DefaultParameters};
+
+// Singleton-like pattern
+static CIRCUIT: Lazy<MockCircuitBuild<DefaultParameters, 2>> = Lazy::new(|| {
+    const SHUFFLE_ROUND_COUNT: usize = 90;
```

Comment:
This should be defined by the wrapper and should be either 10 if the path of the test contains "minimal" and 90 otherwise, i.e. 90 is the default value